### DISCUSSION TITLE
is this a bug for everyone or just for me ?

### DISCUSSION THREAD

**Discussion by sm745052** [community/community#133487] [GitHub Education]
### Hi everyone,

![image.png](https://github.zendesk.com/attachments/token/ulhfDLIioksxmrUMlZbSDkrdn/?name=image.png)
Hi everyone,
I get something like this at many places, like,
1. https://education.github.com/discount_requests/application
2. when I click this
![image.png](https://github.zendesk.com/attachments/token/E6vLmUfOW5azXhzd2jdrrMJer/?name=image.png)

**Comment by ravilovrm** [comment]
i also have it

**Reply by ravilovrm** [reply]
i'm from Russia, Saint-Petersburg
